ci/environment-3.6.yml

```diff
@@ -0,0 +1,16 @@
+name: xhistogram_test_env
+dependencies:
+  - python=3.6
+  - xarray
+  # Dask 2021.3.0 introduced a regression which broke xhistogram.
+  # This regression should be fixed in the next Dask release and we
+  # should remove the dask version pinning below.
+  # see https://github.com/dask/dask/pull/7391
+  # and https://github.com/xgcm/xhistogram/issues/27
+  - dask!=2021.3.0
+  - numpy=1.16
```

Comment:
Instead of testing NumPy 1.16 and 1.17 on Python 3.6, the changes here test NumPy 1.16 on Python 3.6 and NumPy 1.17 on Python 3.7. My _guess_ is this isn't a big deal, but let me know if we should changes what's here